plugins/builds.js

```diff
@@ -119,9 +116,9 @@ exports.plugin = {
                         // inject postMessage into code
                         $('body').append(scriptNode);
                         response = h.response($.html());
-                        response.headers['content-type'] = mime;
-                    } else if (knownMimes.includes(mime)) {
-                        response.headers['content-type'] = mime;
+                    } else {
+                        response.headers['content-disposition'] =
```

Comment:
> Content-Type: image/png
Content-Disposition: inline; filename="picture.png"
Means "This is a PNG image. Please display it unless you don't know how to display PNG images. Otherwise, or if the user chooses to save it, we recommend the name picture.png for the file you save it as".

Reference: https://stackoverflow.com/questions/20508788/do-i-need-content-type-application-octet-stream-for-file-download/20509354#20509354